client/me/billing-history/billing-history-table.jsx

```diff
@@ -37,12 +37,12 @@ class BillingHistoryTable extends React.Component {
 		const { translate } = this.props;
 
 		if ( this.props.sendingBillingReceiptEmail( receiptId ) ) {
-			return translate( 'Emailing Receipt…' );
+			return translate( 'Emailing receipt…' );
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.